src/hooks/useApi.ts

```diff
@@ -66,7 +66,9 @@ export const useApiPatch = ({ path, data, auth }: PatchProps) => {
     window.location.replace('/login');
   }
 
-  return useMutation(path, async () => axios.patch(`${BACKEND_URL}${path}`, data, config).then((res) => res?.data));
+  const apiPatch = ({ data }) => axios.patch(`${BACKEND_URL}${path}`, data, config).then((res) => res?.data);
```

Comment:
`BACKEND_URL` is currently `/api` so it's missing the server host URL and requests fail.